github4s/jvm/src/test/scala/github4s/unit/ApiSpec.scala

```diff
@@ -756,4 +757,145 @@ class ApiSpec
       None)
     response should be('left)
   }
+
+  "Issues >> List" should "return the expected issues when a valid owner/repo is provided" in {
+    val response =
+      issues.list(accessToken, headerUserAgent, validRepoOwner, validRepoName)
+    response should be('right)
+
+    response.toOption map { r ⇒
+      r.result.nonEmpty shouldBe true
+      r.statusCode shouldBe okStatusCode
+    }
+  }
+
+  it should "return an error if invalid data is provided" in {
+    val response =
+      issues.list(accessToken, headerUserAgent, validRepoOwner, invalidRepoName)
+    response should be('left)
+  }
+
+  it should "return an error if no tokens are provided" in {
+    val response =
+      issues.list(None, headerUserAgent, validRepoOwner, validRepoName)
+    response should be('left)
+  }
+
+  "Issues >> Create" should "return the created issue if valid data is provided" in {
+    val response = issues.create(
+      accessToken,
+      headerUserAgent,
+      validRepoOwner,
+      validRepoName,
+      validIssueTitle,
+      validIssueBody,
+      None,
+      List.empty,
+      List.empty)
+    response should be('right)
+
+    response.toOption map { r =>
+      r.statusCode shouldBe createdStatusCode
+    }
+  }
+
+  it should "return an error if invalid data is provided" in {
+    val response = issues.create(
+      accessToken,
+      headerUserAgent,
+      validRepoOwner,
+      invalidRepoName,
+      validIssueTitle,
+      validIssueBody,
+      None,
+      List.empty,
+      List.empty)
+    response should be('left)
+  }
+
+  it should "return an error if no tokens are provided" in {
+    val response = issues.create(
+      None,
+      headerUserAgent,
+      validRepoOwner,
+      validRepoName,
+      validIssueTitle,
+      validIssueBody,
+      None,
+      List.empty,
+      List.empty)
+    response should be('left)
+  }
+
+  "Issues >> Edit" should "return the edited issue if valid data is provided" ignore {
```

Comment:
We should create an issue to fix that part. The root of the problem is the call to [`postData`](https://github.com/47deg/github4s/blob/f6f11910343e9e958d60c9829113b2209e4f39bb/github4s/jvm/src/main/scala/github4s/HttpRequestBuilderExtensionJVM.scala#L58-L58). In that part the request already have set the `PATCH` method but if we take a look to this [method in scalaj-http](https://github.com/scalaj/scalaj-http/blob/master/src/main/scala/scalaj/http/Http.scala#L465) we can see that it's setting the method to `POST`.

One solution could be to replace the `postData` call by:
```diff
-- .postData(body)
++ .postData(d).method(rb.httpVerb.verb)
```

But I've not tested it yet. Sorry about the confusion